### ISSUE TITLE
Have you tried on FUNSD data set

### ISSUE THREAD

**Issue by wendilinplay** [wenwenyu/PICK-pytorch#81]
Hi, thanks for the great work and sharing the code. I'm wondering if you've tried on FUNSD dataset.  FUNSD is always our baseline for benchmarking and another reason I didn't use SROIE is that LayoutLMV2 showed a big jump on FUNSD's performance but not SROIE's.
Anyway, right now with all the default settings I only got 0.60 F1 on FUNSD and I guess there must be something wrong. So I'm reaching out to see if I have the luck that you happen to test on FUNSD and can share the F1, so that I could know where I'm heading.

**Comment by florianbussmann** [comment]
These are my results on FUNSD dataset using PICK with default settings but increased MAX_BOXES_NUM to 220

```
+----------+----------+----------+----------+----------+
| name     |      mEP |      mER |      mEF |      mEA |
+==========+==========+==========+==========+==========+
| answer   | 0.68379  | 0.704373 | 0.693929 | 0.704373 |
+----------+----------+----------+----------+----------+
| header   | 0.467577 | 0.373297 | 0.415152 | 0.373297 |
+----------+----------+----------+----------+----------+
| question | 0.601961 | 0.646316 | 0.62335  | 0.646316 |
+----------+----------+----------+----------+----------+
| overall  | 0.639153 | 0.660393 | 0.6496   | 0.660393 |
+----------+----------+----------+----------+----------+
```

**Comment by wendilinplay** [comment]
Yes, it seems to me that PICK can perform well on small size / sort of
image-rich documents but not text-rich documents.

**Comment by JzjSunshine** [comment]
In my opinion，pick is not suitable for the funsd dataset. PICK is designed to extract key-value pairs in a one-to-one correspondence.
